omero/figure_scripts/Movie_ROI_Figure.py

```diff
@@ -642,68 +639,64 @@ def getTags(name, tagsList, pdList):
             log("Invalid value for scalebar: %s" % str(sb))
             scalebar = None
 
-    overlayColour = (255, 255, 255)
-    if "Scalebar_Colour" in commandArgs:
-        if commandArgs["Scalebar_Colour"] in OVERLAY_COLOURS:
-            r, g, b, a = OVERLAY_COLOURS[commandArgs["Scalebar_Colour"]]
-            overlayColour = (r, g, b)
+    overlay_colour = (255, 255, 255)
+    if "Scalebar_Colour" in command_args:
+        if command_args["Scalebar_Colour"] in OVERLAY_COLOURS:
+            r, g, b, a = OVERLAY_COLOURS[command_args["Scalebar_Colour"]]
+            overlay_colour = (r, g, b)
 
-    roiZoom = None
-    if "Roi_Zoom" in commandArgs:
-        roiZoom = float(commandArgs["Roi_Zoom"])
-        if roiZoom == 0:
-            roiZoom = None
+    roi_zoom = None
+    if "Roi_Zoom" in command_args:
+        roi_zoom = float(command_args["Roi_Zoom"])
+        if roi_zoom == 0:
+            roi_zoom = None
 
-    maxColumns = None
-    if "Max_Columns" in commandArgs:
-        maxColumns = commandArgs["Max_Columns"]
+    max_columns = None
+    if "Max_Columns" in command_args:
+        max_columns = command_args["Max_Columns"]
 
-    showRoiDuration = False
-    if "Show_ROI_Duration" in commandArgs:
-        showRoiDuration = commandArgs["Show_ROI_Duration"]
+    show_roi_duration = False
+    if "Show_ROI_Duration" in command_args:
+        show_roi_duration = command_args["Show_ROI_Duration"]
 
-    roiLabel = "FigureROI"
-    if "Roi_Selection_Label" in commandArgs:
-        roiLabel = commandArgs["Roi_Selection_Label"]
+    roi_label = "FigureROI"
+    if "Roi_Selection_Label" in command_args:
+        roi_label = command_args["Roi_Selection_Label"]
 
     spacer = (width/50) + 2
 
-    print "showRoiDuration", showRoiDuration
-    fig = getSplitView(
-        conn, imageIds, pixelIds, mergedIndexes, mergedColours, width, height,
-        imageLabels, spacer, algorithm, stepping, scalebar, overlayColour,
-        roiZoom, maxColumns, showRoiDuration, roiLabel)
-
-    # fig.show()        # bug-fixing only
+    fig = get_split_view(
+        conn, image_ids, pixel_ids, merged_indexes, merged_colours, width,
+        height, image_labels, spacer, algorithm, stepping, scalebar,
+        overlay_colour, roi_zoom, max_columns, show_roi_duration, roi_label)
 
     if fig is None:
-        logMessage = "No figure produced"
-        log("\n"+logMessage)
-        message += logMessage
+        log_message = "No figure produced"
+        log("\n"+log_message)
+        message += log_message
         return None, message
-    figLegend = "\n".join(logStrings)
+    fig_legend = "\n".join(log_strings)
 
-    # print figLegend    # bug fixing only
-    format = commandArgs["Format"]
+    format = command_args["Format"]
 
-    figureName = "movieROIFigure"
-    if "Figure_Name" in commandArgs:
-        figureName = commandArgs["Figure_Name"]
-        figureName = os.path.basename(figureName)
+    figure_name = "movieroi_figure"
```

Comment:
I will not change it since people might be consuming that name.
This is the name of the generated figure